### PR TITLE
fix: Changed help messages in labels

### PR DESCRIPTION
**Pull request by johannes-b**
Signed-off-by: Johannes <johannes.braeuer@dynatrace.com>

## This PR
Changes labels for the trigger a sequence feature in bridge. 

### Related Issues
Fixes: #7489

### CODE REVIEW

**Comment by johannes-b** [comment]
Will continue, once https://github.com/keptn/keptn/pull/7463 is merged.

**Review by oleg-nenashev — APPROVED** [review]

File: bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.html [hunk 1]
```diff
@@ -272,7 +272,7 @@ <h2 uitestid="ktb-trigger-headline">Trigger an evaluation for {{ selectedService
           >
             <dt-icon name="calendar"></dt-icon>
           </button>
-          <dt-hint>Has to be before end date</dt-hint>
+          <dt-hint>The start date and time of an evaluation that defaults to now if not filled in.</dt-hint>
```
Comment (StackScribe) [inline bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.html:275]:
```suggestion
          <dt-hint>The start date and time of an evaluation that defaults to now if not specified.</dt-hint>
```
Or "provided" would work -- filled in is a little slangy and has that whole dangling preposition problem ;-)

File: bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.html [hunk 2]
```diff
@@ -304,7 +304,7 @@ <h2 uitestid="ktb-trigger-headline">Trigger an evaluation for {{ selectedService
           >
             <dt-icon name="calendar"></dt-icon>
           </button>
-          <dt-hint>Has to be after start date</dt-hint>
+          <dt-hint>The end date and time of an evaluation that has to be after the start date and time.</dt-hint>
```
Comment (StackScribe) [inline bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.html:307]:
```suggestion
          <dt-hint>The end date and time of an evaluation that must be after the start date and time.</dt-hint>
```

**Review by StackScribe — DISMISSED** [review]
lgtm.   A couple trivial word-smithing suggestions for your consideration but I'm approving.

File: bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.html (hunk 2 above)
Comment (johannes-b) [reply bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.html]: Thanks for the hint! I changed it accordingly.

**Review by Kirdock — APPROVED** [review]

**Pull request merged**